community/kernel/src/main/java/org/neo4j/kernel/impl/store/format/standard/StandardFormatSettings.java

```diff
@@ -30,6 +30,8 @@
     static final int PROPERTY_MAXIMUM_ID_BITS = 36;
     public static final int DYNAMIC_MAXIMUM_ID_BITS = 36;
     public static final int LABEL_TOKEN_MAXIMUM_ID_BITS = 32;
+    // TODO decide on correct max bits
```

Comment:
What does the store format of temporal values in #10949 suggest/assume?

Reply:
Nothing yet. There are ~500 different time zones, which we need to be able to address (plus a buffer for future time zones). I will adapt the number accordingly.